Mongo-1-lab/server.js

```diff
@@ -0,0 +1,84 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const mongoose = require('mongoose');
+const Users = require('./model.js');
+
+const STATUS_USER_ERROR = 422;
+const STATUS_SERVER_ERROR = 500;
+const server = express();
+
+server.use(bodyParser.json());
+
+server.post('/users', (req, res) => {
+  const name = req.body.name;
+  console.log('name');
+  if (!name) {
+    res.status(STATUS_USER_ERROR);
+    res.json({ error: 'these are not present' })
+    return;
+  }
+  const users = new Users({ name });
```

Comment:
I would name this something like "user" or "newUser" singular, because that's what you're saving is a single user.